src/frequenz/sdk/microgrid/__init__.py

```diff
@@ -174,53 +174,132 @@
+| B     | 2        | -20kW .. 100kW   | -50kW .. 0kW   | -10kW         |
+| C     | 1        | -20kW .. 0kW     | None           | -20kW         |
+|       |          |                  | target power   | -20kW         |
 
-   Power actually distributed to the batteries: 2500W
+Actor A with the highest priority has the entire battery bounds available to it.
+It sets limited bounds of -20kW .. 100kW.  Actor B can only set powers within
+this range, but actor B overrides the power set by actor A.
 
-2. When the `set_operating_point` actor has made proposals, the bounds available to the
-   regular actors gets shifted, and the final power that actually gets distributed to
-   the batteries is also shifted.
+Actor B tries to limit the bounds of actor C to -50kW .. 0kW, but it can only
+operate in the -20kW .. 100kW range because of bounds set by actor A, so actor C
+sees -20kW .. 0kW.  Actor C proposes a power, so it overrides the power proposed
+by actor B as well.  Because there are no subsequent actors, the power set by
+actor C is used as the target power.
 
-   | actor priority | in op group? | proposed power/bounds | available bounds |
-   |----------------|--------------|-----------------------|------------------|
-   | 3              | No           | 1000, -4000..2500     | -2000..4000      |
-   | 2              | No           | 2500                  | -2000..2500      |
-   | 1              | Yes          | -1000                 | -3000..3000      |
+*Example 2*:
 
-   Power actually distributed to the batteries: 1500W
+| actor | priority | available bounds | request bounds | request power |
+|-------|----------|------------------|----------------|---------------|
+| A     | 3        | -100kW .. 100kW  | -20kW .. 100kW | 50kW          |
+| B     | 2        | -20kW .. 100kW   | -50kW .. 0kW   | -10kW         |
+|       |          |                  | target power   | -10kW         |
+
+When there is no third actor, the second actor's power is the target power.
+
+*Example 3*:
+
+| actor | priority | available bounds | request bounds | request power |
+|-------|----------|------------------|----------------|---------------|
+| A     | 3        | -100kW .. 100kW  | -20kW .. 100kW | 50kW          |
+| B     | 2        | -20kW .. 100kW   | -50kW .. 0kW   | -50kW         |
+|       |          |                  | target power   | -20kW         |
+
+When an actor requests a power that's outside its available bounds, the closest
+available becomes the target power.
+
+#### Shifting the bounds for lower priority actors
+
+When an actor A calls the `propose_power` method with a power, but without any
+bounds (or passes None values for both lower and upper bounds), the lower
+priority actors would see bounds shifted by the power proposed by actor A.
+After lower priority actor B sets a power, it will get shifted back by the power
+set by actor A.  This has the effect of *adding* the powers set by actors A and B.
```

Comment:
I find it a bit confusing to focus on the bounds shifting aspect in this mode instead of focusing on the result (the adding of all power requests). I would write the docs reversing this focus, and saying that when no bounds are requested, the power requests are simply added, and to achieve that, bounds need to be shifted.

Reply:
I had this feeling the whole time I was writing it, but somehow my commitment to the "shifting" narrative was stronger.  Updated.